SequenceBrowser/qSlicerSequenceBrowserModule.cxx

```diff
@@ -57,6 +57,7 @@ class qSlicerSequenceBrowserModulePrivate
   qSlicerSequenceBrowserModulePrivate();
   virtual ~qSlicerSequenceBrowserModulePrivate();
   QTimer UpdateAllVirtualOutputNodesTimer;
+  bool TimerShouldBeActive;
```

Comment:
This variable is set at a few places but its value does not seem to be used anywhere.

Also, the name "should be active" is a bit unclear (why it should or should not be active?).  If we need a variable then its name could be something more explicit, such as "SceneContainsSequenceBrowserNodes" or "VirtualOutputNodeUpdatesNeeded".

Reply:
You're right it didn't turn out to be needed.  I took it out.

It would be cleaner overall to observe the browser nodes and only have the timer on when there's a sequence playing, but that's fairly minor overhead.